disnake/ext/commands/core.py

```diff
@@ -397,13 +398,13 @@ def __init__(
     @property
     def callback(
         self,
-    ) -> CommandCallback[CogT, ContextT, P, T]:
+    ) -> CommandCallback[CogT, Context, P, T]:
```

Comment:
uh, context can be subclassed

Reply:
Yea, but what other solutions are there? `ContextT` isn't bound on the class unlike the other generic parameters, so you'll get a mismatch in the getter and setter since they can refer to different `ContextT` values (which is what this attempts to fix in the first place).

Reply:
hmmm

Reply:
I suppose we could just `# type: ignore` them instead, either way it's not going to perfect 🤔

Reply:
If I'm not entirely misunderstanding, does it not still accept subclasses of `Context` this way, regardless? Sure, `CommandCallback` won't be aware of what subclass it is passed, but I don't think that's important either way.

Reply:
It does work, but pylance shows Context[Unknown]

Reply:
Ah yeah fair, should be able to set `Context[Any]`, then.

Reply:
@shiftinv would you please perform this change?

Reply:
Should hopefully be fixed by 4dd6222db9bcafe515e2f52decd414d3ab34928f, used `Any` and `ContextT` instead of `Context[Any]` as it would've otherwise prevented you from assigning a new callback to `cmd.callback`